modules/37_industry/subsectors/bounds.gms

```diff
@@ -0,0 +1,66 @@
+*** SOF ./modules/37_industry/subsectors/bounds.gms
+
+*** FIXME
+!! $include "./modules/37_industry/subsectors/input/vm_cesIO_scales.inc";
+$ontext
+vm_cesIO.scale(ttot(t2),regi,in_industry_dyn37(in))$(
+                                        NOT tsu(t2) AND vm_cesIO.l(t2,regi,in) )
+  = sum(tttot(t)$( NOT tsu(t) ), vm_cesIO.l(t,regi,in))
+  / (card(ttot) - card(tsu));
+$offtext
+$ifthen.CES_parameters "%CES_parameters%" eq "calibrate"
+  vm_cesIO.scale(t,regi_dyn29(regi),in_industry_dyn37(in))
```

Comment:
I did not know what the .scale suffix does. Maybe adding a short explanation with !! would be useful here.
https://www.gams.com/latest/docs/UG_LanguageFeatures.html#UG_LanguageFeatures_ModelScaling-TheScaleOption

Are the small scales in in_industry_dyn37 determined by 
```suggestion
  !! scaling the industry variables to their value.
  !! The scaling concerns the solve statement only
  !! allows the model to better deal with small values
  vm_cesIO.scale(t,regi_dyn29(regi),in_industry_dyn37(in))   
```
Could this scaling be a source of the infeasibilities you encountered at the 1e-14 level?